setup.py

```diff
@@ -66,7 +67,7 @@
     'invenio-accounts>=1.0.0a10',
     'invenio-pidstore>=1.0.0a7',
     'invenio-jsonschemas>=1.0.0a3',
-    'invenio-files-rest>=1.0.0a1',
+    'invenio-files-rest>=1.0.0a14',
```

Comment:
to use the bump version of fs: https://github.com/inveniosoftware/invenio-files-rest/commit/bd3f87913fd69f85deef653ffc902e141c620d25